integration/presto/src/test/scala/org/apache/carbondata/presto/integrationtest/PrestoTestUsingSparkStore.scala

```diff
@@ -0,0 +1,328 @@
+
+  private val rootPath = new File(this.getClass.getResource("/").getPath
+                                  + "../../../..").getCanonicalPath
+  private val storePath = s"$rootPath/integration/presto/target/store"
+  private val writerPath = storePath + "/presto_spark_db/files"
+  private val sparkStorePath = s"$rootPath/integration/spark/target/spark_store"
+  private val prestoServer = new PrestoServer
+
+  override def beforeAll: Unit = {
+    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.CARBON_WRITTEN_BY_APPNAME,
+      "Presto")
+    val map = new util.HashMap[String, String]()
+    map.put("hive.metastore", "file")
+    map.put("hive.metastore.catalog.dir", s"file://$storePath")
+    prestoServer.startServer("presto_spark_db", map)
+    prestoServer.execute("drop schema if exists presto_spark_db")
+    prestoServer.execute("create schema presto_spark_db")
+  }
+
+  override def afterAll(): Unit = {
+    prestoServer.stopServer()
+    CarbonUtil.deleteFoldersAndFiles(FileFactory.getCarbonFile(storePath))
+     CarbonUtil.deleteFoldersAndFiles(FileFactory.getCarbonFile
+     (s"$sparkStorePath"))
+  }
+
+  def copyStoreContents(tableName: String): Any = {
+    FileUtils.deleteDirectory(new File(writerPath))
+    import java.io.IOException
+    val source = s"$sparkStorePath/$tableName/"
+    val srcDir = new File(source)
+
+    val destination = s"$storePath/presto_spark_db" +
+                      s"/$tableName/"
+    val destDir = new File(destination)
+    try {
+      // Move spark store to presto store path
+      FileUtils.copyDirectory(srcDir, destDir)
+    }
+    catch {
+      case e: IOException =>
+        e.printStackTrace()
+    }
+  }
+
+  test("Test update operations without local dictionary") {
+    prestoServer.execute("drop table if exists presto_spark_db.update_table")
+    prestoServer.execute("drop table if exists presto_spark_db.actual_update_table")
+    prestoServer
+      .execute(
+        "create table presto_spark_db.update_table(smallintColumn smallint, intColumn int, " +
+        "bigintColumn bigint, doubleColumn double, decimalColumn decimal(10,3), " +
+        "timestampColumn timestamp, dateColumn date, " +
+        "stringColumn varchar, booleanColumn boolean ) with(format='CARBON') ")
+    prestoServer
+      .execute(
+        "create table presto_spark_db.actual_update_table(smallintColumn smallint, intColumn int," +
+        " " +
+        "bigintColumn bigint, doubleColumn double, decimalColumn decimal(10,3), " +
+        "timestampColumn timestamp, dateColumn date, " +
+        "stringColumn varchar, booleanColumn boolean ) with(format='CARBON') ")
+    copyStoreContents("update_table")
+    copyStoreContents("actual_update_table")
+    assert(prestoServer
+      .executeQuery("SELECT * FROM presto_spark_db.update_table").equals(prestoServer
+      .executeQuery("SELECT * FROM presto_spark_db.actual_update_table")))
```

Comment:
I think each test case can copy its content. After copy you can call drop table itself.

Reply:
you are suggesting to drop immediately after copying not just for this but remaining other test cases too?

Reply:
I mean for spark testcase, you are not calling drop table at all, so it will never be dropped. so, if you copy one by one. you can call drop table also in spark testcase

Reply:
Dropped all tables in afterAll()